packages/suite/src/hooks/suite/useDevice.ts

```diff
@@ -7,12 +7,15 @@ export const useDevice = () => {
     const device = useSelector<AppState, AppState['suite']['device']>(state => state.suite.device);
     const locks = useSelector<AppState, AppState['suite']['locks']>(state => state.suite.locks);
 
-    const isLocked = useCallback(() => {
-        if (!device || !device.connected) return true;
-        if (locks.includes(SUITE.LOCK_TYPE.DEVICE) || locks.includes(SUITE.LOCK_TYPE.UI))
-            return true;
-        return false;
-    }, [device, locks]);
+    const isLocked = useCallback(
+        (ignoreDisconnectedDevice?: boolean) => {
+            if ((!device || !device.connected) && !ignoreDisconnectedDevice) return true;
```

Comment:
This can be simplified using optional chaining: `!device?.connected` 
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
yup, there are certainly a lot of places that would benefit from optional chaining since is rather new feature, maybe we could rewrite them all in one PR, but I don't find it necessary as usage of both is fine with me (I only added `ignoreDisconnectedDevice` to the condition).